src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/CellUpdate.cs

```diff
@@ -197,6 +196,18 @@ private void ProcessBooleanCell(string valueAsString)
             ValueAsString = Value.ToString();
         }
 
+        private void ProcessNullValue()
+        {
+            // Make sure that nulls are allowed if we set it to null
+            if (!Column.AllowDBNull.HasTrue())
+            {
+                throw new InvalidOperationException(SR.EditDataNullNotAllowed);
```

Comment:
This would be caught in the edit/updateCell handler and the exception message would be returned to the user via an error.